IPython/core/tests/test_magic_arguments.py

```diff
@@ -116,3 +116,21 @@ def test_magic_arguments():
     assert_equal(real_name(foo), 'foo')
     assert_equal(foo(None, ''), argparse.Namespace(foo=None))
     assert hasattr(foo, 'has_arguments')
+
+
+@magic_arguments()
+@argument("filepath", type=str, nargs=1, help="An argument with possible spaces")
+def magic_one_arg(args):
+    return parse_argstring(magic_foo1, args)
```

Comment:
I think you meant..?

```suggestion
    return parse_argstring(magic_one_arg, args)
```

Reply:
good catch, of course! that was my copy-pasta!